baggage/baggage_test.go

```diff
@@ -469,6 +470,13 @@ func TestBaggageParse(t *testing.T) {
 			in:   tooManyMembers,
 			err:  errMemberNumber,
 		},
+		{
+			name: "percent-encoded octet sequences do not match the UTF-8 encoding scheme",
+			in:   "k=aa%ffcc",
+			want: baggage.List{
+				"k": {Value: "aa�cc"},
+			},
+		},
```

Comment:
It looks that we are missing handling invalid  UTF-8 for property values.

```suggestion
		{
			name: "percent-encoded octet sequences do not match the UTF-8 encoding scheme",
			in:   "k=aa%ffcc;p=d%fff",
			want: baggage.List{
				"k": {
					Value: "aa�cc",
					Properties: []baggage.Property{
						{Key: "p", Value: "d�f",, HasValue: true},
					},
				},
			},
		},
```

Reply:
done